mu-plugins/central-hub/tests/phpunit/unit/config-store/_theStore.php

```diff
@@ -66,11 +69,13 @@ public function test_should_return_all_stored_configs_when_no_key_or_configs() {
 			],
 		];
 		foreach ( $configs as $store_key => $config ) {
-			_the_store( $store_key, $config );
+			loadConfig( $store_key, $config );
 		}
 
 		// Should return the all stored configs.
 		$this->assertSame( $configs, _the_store() );
+
+		self::empty_the_store_by_keys( [ 'foo', 'bar', 'baz' ] );
```

Comment:
@rgadon107 The reason why this test did not empty the keys is because we haven't tested that functionality for `_the_store`. This test class is for the store. Right? First we test that we can store. Then we incrementally test different parts of the behavior. But we can't jump straight to emptying the store until we know that emptying works.